src/Microsoft.NET.Workloads/Microsoft.NET.Workloads.csproj

```diff
@@ -14,7 +14,8 @@
     <PackageId>$(MSBuildProjectName).$(SDKFeatureBand)</PackageId>
     <!-- Adds a README.md to the NuGet package. -->
     <!-- See: https://devblogs.microsoft.com/nuget/add-a-readme-to-your-nuget-package/ -->
-    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <ReadmeFile>$(ArtifactsNonShippingPackagesDir)README.md</ReadmeFile>
+    <ReadmeFileTemplate>README.md</ReadmeFileTemplate>
```

Comment:
Why was the `PackageReadmeFile` property removed? This is the mechanism that NuGet uses to put the readme into the package during the Pack target. The value in this property is the path to the readme in the package itself. I had put an example of this in the project-system code here: https://github.com/dotnet/project-system/pull/7440/files#diff-e027b89f63ba0194b63411497c368204b9fc2a17b915a552139abf33f6d82a33R10-R27

Reply:
See my comment above. I was getting NU5039 when including that property and I found the include put the readme file in the package anyway (at least locally). I should check the test build I did just to be sure.